packages/shorebird_cli/lib/src/commands/patch/patch_command.dart

```diff
@@ -82,7 +82,7 @@ of the iOS app that is using this module.''',
       )
       ..addFlag(
         'codesign',
-        help: 'Codesign the application bundle.',
+        help: 'Codesign the application bundle (iOS only).',
```

Comment:
Maybe we should add `(iOS only)` to the exportOptionsPlist as well?

Reply:
Done